spec/controllers/task_managements_controller_spec.rb

```diff
@@ -38,7 +38,7 @@
       it "assigns tasks given to taskee" do
         @user.update_attribute(:user_type, "taskee")
         get :index
-        expect(assigns(:tasks)).to eql @user.tasks_given.to_a
+        expect(assigns(:tasks)).to eql @user.tasks_given.where(paid: true).to_a
```

Comment:
You can also reuse the paid task scope here and in other places where we have `where(paid: true)`